src/dftbp/common/file.F90

```diff
@@ -218,13 +218,16 @@ subroutine TFileDescr_connectToFile(this, file, options, mode, ioStat, ioMsg)
     !! * "w+": readwrite (file created or truncated if it already exists)
     !! * "a": append-write (file opened if exists, otherwise created, positioned at the end)
     !! * "a+": append-read/write (file opened if exists, otherwise created, positioned at the end)
-    !! The values above can be followed by "t" for text/unformatted mode (default) or "b" for
+    !! The values above can be followed by "t" for text/formatted mode (default) or "b" for
     !! binary/unformatted mode.
     !!
     !! When arguments options and mode are both specified, the resulting options are determined by
     !! applying options first and then set the fields which mode manipulates.
     character(*), optional, intent(in) :: mode
 
+    !> If false, blocks writing to files for all processes other than the lead (default: false).
```

Comment:
```suggestion
    !> If false or absent, block writing to files by all processes other than the lead (default: false).
```
Also, possibly need to make this clear that this is the global lead for MPI_COMM_WORLD, as per discussion about library use on a sub-group.